mne/io/egi/events.py

```diff
@@ -60,6 +72,12 @@ def _read_mff_events(filename, sfreq, nsamples):
         for event in orig[xml][2:]:
             event_start = _ns2py_time(event['beginTime'])
             start = (event_start - start_time).total_seconds()
+            # adjust the start to account for epochs
+            epoch = [i[0] for i in enumerate(epoch_times)
+                     if (i[1][0] <= start and i[1][1] >= start)]
+            if len(epoch) == 0:
+                continue
```

Comment:
I add a comment above the continue that:
```
# event not within the recording periods
```
but is this possible for an event to be saved but not within the recording period? If that is not a normal scenario it might be worth throwing a warning here. If `len(epoch) == 0` should not happen, then maybe an error?